lib/outputs/outputs.py

```diff
@@ -49,13 +49,14 @@ def report(check_findings, output_options, audit_info):
         # csv_fields = []
         file_descriptors = {}
         if output_options.output_modes:
-            # We have to create the required output files
-            file_descriptors = fill_file_descriptors(
-                output_options.output_modes,
-                output_options.output_directory,
-                output_options.output_filename,
-                audit_info,
-            )
+            if not hasattr(audit_info, "is_azure"):
```

Comment:
If this is only for AWS, please include a comment to mark it because we will need to address it in the future. Also you can check if the `audit_info` is a class of `AWS_Audit_Info` with `isinstance(audit_info, AWS_Audit_Info)`, you don't need to include more fields.

Reply:
Changed